contrib/dockerd-rootless.sh

```diff
@@ -94,6 +94,6 @@ else
 	[ $_DOCKERD_ROOTLESS_CHILD = 1 ]
 	# remove the symlinks for the existing files in the parent namespace if any,
 	# so that we can create our own files in our mount namespace.
-	rm -f /run/docker /run/xtables.lock
+	rm -f /run/docker /run/containerd /run/xtables.lock
```

Comment:
This I assume is in the new mount namespace, and therefore the tmpfs is not from the host?

Reply:
Yeah, and it's not `rm -r`, so the potential impact if this happens to have a more "real" `/run/containerd` is low (it'll probably fail).

Reply:
Yes, it is a new mount namespace. `/run/containerd` being removed is a symlink to the root-owned `/run/containerd` in the parent namespace. So we can safely remove it without privileges.